cogs/welcome/welcome.py

```diff
@@ -30,6 +35,9 @@
     KEY_TITLE: "Welcome!",
     KEY_MESSAGE: "Welcome to the server! Hope you enjoy your stay!",
     KEY_IMAGE: None,
+    "KEY_GREETINGS": [],
```

Comment:
It looks like this list contains a 2-element tuple containing the greeting name and the message. I think it would be better to have this as a dictionary, with the key being the greeting name, and the value being the message. This way, the lookup is basically "free": just check for the existent of the key (which is the greeting name in this case).

Reply:
Switched to dictionary